<ui version="4.0" >
 <class>OverviewSelectBase</class>
 <widget class="QDialog" name="OverviewSelectBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Overview plot selection</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="1" column="0" >
        <spacer name="spacer3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" >
        <spacer name="spacer4" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="StatusLabel" >
           <property name="text" >
            <string>Select test status:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="StatusComboBox" >
           <item>
            <property name="text" >
             <string>all</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="quantLabel" >
           <property name="text" >
            <string>Select overview plot:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer1" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>31</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="PlotComboBox" >
           <item>
            <property name="text" >
             <string>Avg. threshold</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Threshold disp.</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Avg. noise</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Analog good pixels</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Good bumps</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Digital scan info</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="SystestChkBox" >
       <property name="text" >
        <string>System test mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="SystestGrpBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="title" >
        <string>System test options</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1" >
              <property name="text" >
               <string>Available datasets:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="AvailableComboBox" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text" >
                <string>my dummy label for a threhsold scan: 2000/01/01 00:00:00</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="textLabel1_2" >
              <property name="text" >
               <string>You selected a scan of type:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="ScanLabel" >
              <property name="text" >
               <string>none</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel3" >
              <property name="text" >
               <string>Plot options:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer7" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="OptionsComboBox" >
              <item>
               <property name="text" >
                <string>threshold dispersion</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel4" >
              <property name="text" >
               <string>Referenced to:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer8" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="ReferenceComboBox" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>260</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text" >
                <string>my dummy label for a threhsold scan: 2000/01/01 00:00:00</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PlotBtn" >
       <property name="text" >
        <string>Plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
